create-web3signer-secrets.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
is this script a good place to execute the go util to decrypt the keys and upload to Vault?

Reply:
Yes, added this ticket - https://app.zenhub.com/workspaces/obol-workspace-648c1170ee94656741353327/issues/gh/obolnetwork/charon-k8s-distributed-validator-cluster/121